### DISCUSSION TITLE
ERROR: DeePMD-kit Error: TensorFlow Error:  INVALID_ARGUMENT: 2 ROOT errors found, INVALID_ARGUMENT:  required broadcatsble shapes

### DISCUSSION THREAD

**Discussion by sddtju** [deepmodeling/deepmd-kit#3683] [Q&A]
I get an error when I train out a model in the GPU version of deepmd-kit and run MD with it in lammps
LAMMPS (23 Jun 2022 - Update 1)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
Loaded 1 plugins from /opt/software/anaconda3/envs/deepmd/lib/deepmd_lmp
Loaded 0 plugins from 
pair_style deepmd R134a.pb
Summary of lammps deepmd module ...
  >>> Info of deepmd-kit:
  installed to:       /opt/software/anaconda3/envs/deepmd
  source:             v2.2.0
  source branch:       HEAD
  source commit:      6cdc5bff
  source commit at:   2023-03-14 13:16:23 +0800
  surpport model ver.:1.1 
  build variant:      cuda
  build with tf inc:  /opt/software/anaconda3/envs/deepmd/include;/opt/software/anaconda3/envs/deepmd/include
  build with tf lib:  /opt/software/anaconda3/envs/deepmd/lib/libtensorflow_cc.so
  set tf intra_op_parallelism_threads: 0
  set tf inter_op_parallelism_threads: 0
  >>> Info of lammps module:
  use deepmd-kit at:  /opt/software/anaconda3/envs/deepmdpair_coeff * *
ERROR: DeePMD-kit Error: TensorFlow Error: INVALID_ARGUMENT: 2 root error(s) found.
  (0) INVALID_ARGUMENT: required broadcastable shapes
	 [[{node mul_3}]]
	 [[o_energy/_31]]
  (1) INVALID_ARGUMENT: required broadcastable shapes
	 [[{node mul_3}]]
0 successful operations.
0 derived errors ignored. (/home/conda/feedstock_root/build_artifacts/libdeepmd_1678814076702/work/source/lmp/pair_deepmd.cpp:471)
Last command: minimize 1.0e-7 1.0e-7 10000 10000     
<img width="611" alt="微信图片_20240418204701" src="https://github.com/deepmodeling/deepmd-kit/assets/167435875/a6735004-f30e-4a9b-b611-9420ba93f115">

**Comment by njzjz** [comment]
v2.2.0 is known to have the issue #2398.
